### PR TITLE
pkg/{cgroups,topology}: move linux-specific functions

### PR DESCRIPTION
**Pull request by elezar**
This change moves linux-specific functions to `_linux.go` source files and adds stub implementations for other platforms.

### CODE REVIEW

File: pkg/cgroups/cgroupid_linux.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package cgroups
+
+import "encoding/binary"
```
Comment (askervin) [inline pkg/cgroups/cgroupid_linux.go:3]: missing import "golang.org/x/sys/unix".

Comment (elezar) [reply pkg/cgroups/cgroupid_linux.go:3]: Yes, you're right. I was building locally on a `darwin` machine. Let me do so in a container too.

**Review by klihub — APPROVED** [review]
LGTM

**Review by askervin — APPROVED** [review]
LGTM

**Pull request merged**